ydb/core/kqp/compile_service/kqp_compile_actor.cpp

```diff
@@ -372,6 +372,14 @@ class TKqpCompileActor : public TActorBootstrapped<TKqpCompileActor> {
         PassAway();
     }
 
+    void FillPreparedQuery(std::unique_ptr<NKikimrKqp::TPreparedQuery> preparingQuery, NKikimrKqp::EQueryType queryType) {
```

Comment:
We fill `KqpCompileResult` in this function. Maybe it's better to name it `FillCompileResult`?